src/app/pages/details/common/get-this-title-files/options.js

```diff
@@ -223,7 +224,11 @@ export function KindleOption({model}) {
         <SimpleLinkOption
             link={model.kindleLink} icon={['fab', 'amazon']} text="Download for Kindle"
             data-track="Kindle"
-        />
+        >
+            <div class="disclaimer">
+                As an Amazon Associate we earn from qualifying purchases
```

Comment:
@RoyEJohnson is this not being provided as part of the buy print link stuff?

Reply:
https://buyprint.openstax.org/prealgebra-2e.json

Reply:
Buyprint is for print copies. Kindle is distinct. It might be worth discussing whether Buyprint should do Kindle as well, but for now it's a link in the CMS.